cns/service/main.go

```diff
@@ -872,6 +872,11 @@ func main() {
 		}
 	}
 
+	if cnsconfig.BackendNICEnabled {
```

Comment:
I think we should enable this by default as there is no way of know when to and when not to. And if MTPNC sends this data we should process it.

Reply:
If we perform this during add call, it can lead to delay in ipconfigsresponse. Powershell commands sometimes takes longer than expected. Doing this over the boot time reduces this delay.

Reply:
I have removed the need for config, I have moved this function to windows platform, it will be no op for linux. I will bucket this under cns swiftv2 mode hence it will get executed only for swift v2 and windows platform.